### PR TITLE
Mail!

### PR DESCRIPTION
**Pull request by DebugOk**
## About the PR
Adds mail!

## Why / Balance
Because mail is cool

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Mail has been reimplemented

### CODE REVIEW

**Comment by DebugOk** [comment]
It do be ready for review

**Comment by DEATHB4DEFEAT** [comment]
guh

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetSteel1:
+            min: 1
+            max: 1
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: ApcPowerReceiver
+    powerLoad: 1000 # TODO if we keep this make it spike power draw when teleporting
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml:48]: Spike the power draw

Comment (DebugOk) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml:48]: Cargo on their way to cause brownouts just by having a mail teleporter

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml:48]: :trollface:

Comment (DebugOk) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml:48]: I'm not 100% sure how to go about spiking the draw properly though

Comment (Colin-Tel) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml:48]: Are we waiting on this to merge it in? Didn't want to jump the gun or anything.

Comment (DebugOk) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml:48]: I'm honestly not sure how to properly implement this, so we could just leave it as a todo

**Comment by DebugOk** [comment]
Well then, lets hope Death didn't have any other things to point out :blunt:

**Pull request merged**